tests/console/yast2_clone_system.pm

```diff
@@ -44,22 +44,19 @@ sub run {
 
     unless (is_opensuse) {
         my $devel_repo = get_required_var(is_sle('>=15') ? get_repo_var_name("MODULE_DEVELOPMENT_TOOLS") : 'REPO_SLE_SDK');
-        zypper_call "ar -c $utils::OPENQA_FTP_URL/" . $devel_repo . " devel-repo";
+        my $uri        = (is_staging) ? $devel_repo : "$utils::OPENQA_FTP_URL/" . $devel_repo;
+        zypper_call "ar -c -G $uri devel-repo";
     }
+    zypper_call '--gpg-auto-import-keys ref' unless (is_staging);
```

Comment:
when there is staging (sle/openSUSE) there is no ref of the repo, Downloading metadata should be done always or I missing something?